docs/troubleshooting-guide.md

```diff
@@ -57,6 +57,33 @@ If `tglogutil repair` succeeds, tsurugidb can be started by `tgctl start`.
 For more information of `tglogutil repair`, please see the following documentation.
 - https://github.com/project-tsurugi/limestone/blob/master/docs/tglogutil-repair-man.md
 
+### Too many open files
+
+ `tgctl start` command fails with the following error message:
```

Comment:
Normally it tales a long time to start up, and this error occurs after tgctl times out for 10 secs.
So, I think it would be better to write something like "when tgctl start fails, the server log shows the following messages."

Reply:
Revised on https://github.com/project-tsurugi/tsurugidb/pull/79/commits/ff286fbe37b33b9907f7e746f23c3f3bb4252a72